containers/gcp/content/run-with-gce.sh

```diff
@@ -107,45 +106,21 @@ else
   INSTANCE_PREFIX="datalab-${ESCAPED_USER_EMAIL}"
 fi
 
-INSTANCE=`gcloud compute instances list --project "${PROJECT}" --zone "${ZONE}" --regex "${INSTANCE_PREFIX}-[0-9]*" --limit 1 --format "value(name)"`
+INSTANCE=`gcloud compute instances list --project "${PROJECT_ID}" --zone "${ZONE}" --regex "${INSTANCE_PREFIX}-[0-9]*" --limit 1 --format "value(name)"`
```

Comment:
The new image has to be run in a container that has a volume mapping the host's gcloud config directory to the container's gcloud config directory (so it uses the hosts auth configuration).

You're absolutely right that I need to document that. I also forgot to add a README to this directory. I'll add that file and put the details in there.